qutebrowser/browser/webkit/webkitelem.py

```diff
@@ -306,6 +306,9 @@ def _move_text_cursor(self):
             self._tab.caret.move_to_end_of_document()
 
     def _click_editable(self, click_target):
+        # set UserInteracted in focustools so our focus event goes through
+        self._elem.evaluateJavaScript(
+            'window._qutebrowser.focustools.setUserInteracted();')
```

Comment:
Like mentioned above, this should be some kind of function in the API. You probably don't need the comment anymore then either.